Resources/Private/Templates/Backend/SocialImageWizard.html

```diff
@@ -1,6 +1,4 @@
-<html xmlns:f="http://typo3.org/ns/TYPO3/CMS/Fluid/ViewHelpers"
-      xmlns:core="http://typo3.org/ns/TYPO3/CMS/Core/ViewHelpers">
-
+<html xmlns:f="http://typo3.org/ns/TYPO3/CMS/Fluid/ViewHelpers" xmlns:core="http://typo3.org/ns/TYPO3/CMS/Core/ViewHelpers" data-namespace-typo3-fluid="true">
```

Comment:
Actually it's nice to have each attribute on its own line since that reduces Git diffs in the future.

Reply:
I think we can adjust this at a later point, for now, I would like to keep it like that for consistency reasons